Guardian.Backend/Guardian/Controllers/GameController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Guardian.Service.Features.CustomerFeatures.Queries;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using System.Threading.Tasks;
+using Guardian.Domain.Entities;
+using System;
+using Guardian.Persistence;
+
+namespace Guardian.Controllers
+{
+
+    [Authorize]
+    [ApiController]
+    [Route("api/v{version:apiVersion}/Games")]
+    [ApiVersion("1.0")]
+    public class GameController : ControllerBase
+    {
+        private IMediator _mediator;
+        protected IMediator Mediator => _mediator ??= HttpContext.RequestServices.GetService<IMediator>();
+
+        [HttpGet]
+        [Route("")]
+        public async Task<IActionResult> GetAll()
+        {
+            var result = await Mediator.Send(new GetAllGamesQuery());
```

Comment:
należy pobrać jako parametr paginacje i przekazać to jako argument do GetAllGamesQuery